University of Ljubljana - Faculty of civil and geodetic engineering (slovenian)

```diff
@@ -0,0 +1,1519 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded" default-locale="sl-SI">
+  <info>
+    <title>University-of-Ljubljana-Faculty-of-Civil-and-Geodetic-Engineering-(Slovenian)</title>
```

Comment:
1. please remove the dashes from the title here
2. Please add Slovenscina in the original language as shown here: https://www.zotero.org/styles?q=id%3Auniverza-na-primorskem-fakulteta-za-vede-o-zdravju-apa
3. please remove "slovenian" from the filename
4. your self link in line 7 needs to match the filename (without the .csl)